iterator_practice.cpp

```diff
@@ -1,16 +1,13 @@
 #include <iostream>
-#include <iterator> // for iterator
 #include <vector> // for vectors 
 
-using namespace std;
-
-int main()
-{
-    vector<int> ar = {2,4,6,8,10};
-    vector<int> ::iterator ptr;
-    cout << "My vector elements are: ";
-    for (ptr = ar.begin(); ptr < ar.end (); ptr++)
-        cout << *ptr<< " ";
-
-    return 0; 
-}
\ No newline at end of file
+int main() {
+  const std::vector<int> evens{2,4,6,8,10};
+  std::cout << "My vector elements are: ";
+  for (auto it = evens.begin(); it < evens.end(); ++it) {
+    std::cout << *it << " ";
+  }
+  std::cout << "\n";
```

Comment:
It's a bit nicer to end your output with a new line, so in the terminal when you run it you get a fresh line afterward